cocos/ui/scroll-view.ts

```diff
@@ -880,11 +901,16 @@ export class ScrollView extends ViewGroup {
      * @zh
      * 获取当前视图内容的坐标点。
      *
-     * @returns - 当前视图内容的坐标点.
+     * @returns - current content position.
+     * @deprecated Since 3.0, getContentPosition is deprecated.
      */
     public getContentPosition () {
+        return this._getContentPosition();
+    }
+
+    private _getContentPosition () {
```

Comment:
Even if you deprecated g(s)etContentPosition, there is no meaning to add a _g(s)etContentPosition